terraform/appflask.py

```diff
@@ -761,9 +761,8 @@ def calico_nodes_view():
             calico_nodes.append({"hostname": hostname, "ip": ip,
                                 "ipv6": ipv6, "natip": natip, "rack_id": rack_id})
             if turbo.can_stream():
```

Comment:
You have it here (I am referring to the next comment) but maybe is better to put it all in one place?

Reply:
You mean inside right